### ISSUE TITLE
compiling (and starting) branch dotdevelop (based on last commit monodevelop/master )

### ISSUE THREAD

**Issue by lytico** [dotdevelop/community#10]
in https://github.com/dotdevelop/dotdevelop, branch dotdevelop (excuse my narrow fantasy for naming) is a compiling fork of monodevelop/master (last commit).
It starts, and then stops with an (ide-visible) error-dialog. on my own branch (lytico_develop) it even shows the main window.

It could be a base experimenting / refactoring etc.
it uses forks (named dotdevelop) of several subprojects, eg. Xwt, vs-editor-ide, etc.
details see: https://github.com/dotdevelop/dotdevelop/blob/dotdevelop/.gitmodules

try like this (everything on linux):

```
git clone -b dotdevelop https://github.com/dotdevelop/dotdevelop.git
cd dotdevelop/
./configure --profile=gnome
make
cd main
msbuild -r -p:Configuration=DebugGnome
```
then open in an ide of your choice (eg. monodevelop 7.8, rider also does), and choose configuration

DebugGnome

use in Solution / Core
MonoDevelop.Startup

as startup project.

looks like this:
debugging monodevelop (version 8.6 Preview) over monodevelop (version 7.8.4 Build 2):

![grafik](https://user-images.githubusercontent.com/1479980/80649187-d6ecc480-8a71-11ea-8ec2-386519700c02.png)

**Comment by lextm** [comment]
Did you get a working editor from there? If not, then 7.x might be a more suitable starting point.

**Comment by lytico** [comment]
no, not at all. all editor-related stuff are stubs. it basically just compiles, and to my surprise even shows the ide.

**Comment by lextm** [comment]
@lytico the editor is just a small piece of the IDE, so not surprising you can compile and launch most of the parts. I did the same on Windows a while ago, but the 8.x editor was broken.

However, code editor is still the most critical part of an IDE, without a working editor 8.x branches are less useful.

**Comment by lytico** [comment]
no question, i agree. one spike could be port 7.8 editor into 8.6 branch.

another way is backport usefull parts of 8.6 to 7.8.
if you have an 7.8 version, i suppose to branch it into our repository,
eg. create a branch dotdevelop_7.8

there are other solutions to have an editor too, eg. the vs-code-editor.

**Comment by lytico** [comment]
in every case, we need an ide-version based on gtksharp, running on dotnetcore ff.  with gtk-sharp 2 we are stucked on .net framework.
